apps/site/src/app/(main)/guest-login/GuestLogin.tsx

```diff
@@ -20,7 +21,10 @@ async function GuestLogin({
 			<h1 className="font-display text-3xl md:text-5xl mb-20">
 				Enter Passphrase
 			</h1>
-			<GuestLoginVerificationForm />
+			<GuestLoginVerificationForm
+				email={searchParams?.email}
+				return_to={searchParams?.return_to}
```

Comment:
Suggestion: since we have more of these now, might be simpler to unspread them at the beginning of the function and shorten the specification also on line 16, e.g.
```tsx
const { email, return_to } = searchParams;
```

Reply:
I will go through and make these changes elsewhere too.